cppForSwig/Makefile

```diff
@@ -41,7 +41,7 @@ PYTHON_INCLUDES=$(shell $(PYVER)-config --includes )
 endif
 
 CPPFLAGS += $(ARMORY_CPPFLAGS) -Icryptopp -Imdb -DUSE_CRYPTOPP -D__STDC_LIMIT_MACROS
-LDLIBS += -lpthread -Lmdb
+LDLIBS += -Wl,--no-as-needed -lpthread -lrt -Lmdb
```

Comment:
I merged a slightly different version of this into 0.93-bugfix. Can you revise this to match what I submitted?